tests/foreman/ui_airgun/test_role.py

```diff
@@ -0,0 +1,317 @@
+"""Test class for Roles UI
+
+:Requirement: Role
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from airgun.session import Session
+from nailgun import entities
+from navmazing import NavigationTriesExceeded
+from pytest import raises
+
+from robottelo.datafactory import gen_string
+from robottelo.decorators import fixture, tier2, upgrade
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def module_loc():
```

Comment:
Should we make this one reserved and selected by default for `module_user` (similar to `module_org`)?

Reply:
that is going to be out of scope for current PR. I tried to go same way we had for organizations, but it doesn't seem that id=1 is always default for location. For my specific server it was id=2. It can be a default number for all servers, but it is time consuming test procedure as all modules are affected by such change